### PR TITLE
Added the yaml logo to README file

### LINKED ISSUE #151: Add the YAML logo to the readme.md file
Add the YAML logo to the readme.md file

image

### PR DESCRIPTION
**Pull request by zmusaddique**
## Purpose

The purpose of this Pull Request is to fix #151

## Goals

Yaml logo links to yaml tutorial by Tutorialspoint

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Send me a screenshot @zmusaddique

**Comment by zmusaddique** [comment]
image

This is how the readme looks

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Send me a screenshot after resizing the icon @zmusaddique

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
@zmusaddique Screenshot pls

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 <a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a>
 <a href="https://www.w3schools.com/bootstrap/"><img src="https://img.icons8.com/color/48/000000/bootstrap.png"/><a>
 <a href="https://www.w3schools.com/python/"><img src= "https://img.icons8.com/color/48/000000/python.png"/><a>
+<a href="https://www.tutorialspoint.com/yaml/index.htm"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5a/Official_YAML_Logo.svg/48px-Official_YAML_Logo.svg.png"/><a>
```
Comment (YoshithaRathnayake) [inline README.md:18]: Add the YAML logo after the JS icon @zmusaddique

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zmusaddique) [reply README.md:18]: OK

Comment (zmusaddique) [reply README.md:18]: Are you happy with this?

Comment (zmusaddique) [reply README.md:18]: image

Is this fine?

**Review by YoshithaRathnayake — APPROVED** [review]
Good Job !
Thank You @zmusaddique

**Comment by zmusaddique** [comment]
> Good Job ! Thank You @zmusaddique

You're welcome @YoshithaRathnayake 
Thanks for guiding me along !

**Comment by YoshithaRathnayake** [comment]
> > Good Job ! Thank You @zmusaddique
> 
> You're welcome @YoshithaRathnayake Thanks for guiding me along !

Any time @zmusaddique

**Pull request merged**